src/oas3/__tests__/bundle.test.ts

```diff
@@ -1013,10 +1013,6 @@ describe('bundleOas3Service', () => {
       const res = bundleOas3Service({
         document: {
           openapi: '3.0.0',
-          info: {
-            version: '0.0.1',
-            title: 'ref source spec',
-          },
```

Comment:
I removed these `info` sections from all the tests, since they don't affect anything.